### PR TITLE
Use specified bootstrapInterval to adjust start time in holtWinters* functions

### PR DESCRIPTION
**Pull request by carrieedwards**
This PR fixes an issue that was found with the holtWinters* functions. Previously, the code was adjusting the start time of these functions by 7 days, which is the default value for the bootstrapInterval parameter. However, if a different bootstrapInterval was specified, when the series was retrieved via GetSeriesArg, the from value in the request was altered by subtracting the bootstrapInterval, and the previously fetched data was not matching the request due to the altered start time. Thus, nil results were being returned. This PR fixes this by factoring in the specified bootstrapInterval when adjusting the start time for holtWinters* Functions, and only using the default of 7d if no bootstrapInterval is specified in the query.

### CODE REVIEW

File: pkg/parser/parser.go [hunk 1]
```diff
@@ -196,10 +196,6 @@ func (e *expr) Metrics(from, until int64) []MetricRequest {
 			}
 
 			return r2
-		case "holtWintersForecast", "holtWintersConfidenceBands", "holtWintersAberration":
-			for i := range r {
-				r[i].From -= 7 * 86400 // starts -7 days from where the original starts
-			}
```
Comment (npazosmendez) [inline pkg/parser/parser.go:202]: Can't this from/to adjustment be done here, beforehand? Otherwise we would be doing one fetch unnecessarily

Comment (carrieedwards) [reply pkg/parser/parser.go]: That is also a solution. Originally I had gone the route of re-fetching in order to more closely resemble Graphite web's code, as I have found the adjusting before fetch time to be confusing for debugging at times. But I will re-implement it this way to save the extra fetch.

File: expr/functions/holtWintersAberration/function.go [hunk 2]
```diff
@@ -36,7 +36,9 @@ func (f *holtWintersAberration) Do(ctx context.Context, e parser.Expr, from, unt
 		return nil, err
 	}
 
-	args, err := helper.GetSeriesArg(ctx, e.Arg(0), from-bootstrapInterval, until, values)
+	// Note: the start time for the fetch request is adjusted in expr.Metrics() so that the fetch request
+	// for the data has a from time that factors in the bootstrapInterval
+	args, err := helper.GetSeriesArg(ctx, e.Arg(0), from, until, values)
```
Comment (npazosmendez) [inline expr/functions/holtWintersAberration/function.go:41]: Is this change needed? I would expect the `from-bootstrapInterval` to be still needed, since that's what the fetched request looks like

**Review by npazosmendez — APPROVED** [review]
